superset-frontend/src/explore/components/DatasourcePanel.tsx

```diff
@@ -204,7 +206,12 @@ export default function DataSourcePanel({
             </div>
             {metricSlice.map(m => (
               <LabelContainer key={m.metric_name} className="column">
-                <MetricOption metric={m} showType />
+                <DatasourcePanelDragWrapper
+                  metricOrColumnName={m.metric_name}
+                  type={DatasourcePanelDndType.METRIC}
+                >
+                  <MetricOption metric={m} showType />
```

Comment:
What do you think about using a `cursor: move` instead of default here?

Reply:
sorry, what is `cursor: move` on this?

Reply:
I mean that here we could use `cursor: move` to indicate that the label is draggable 
![image](https://user-images.githubusercontent.com/15073128/108846169-8d5ffe00-75de-11eb-9b73-b3991316634c.png)

Reply:
All right, I will add indicator style on this component and drop component.

Reply:
The style `pointer: move` on the datasource panel looks strange. the UI behavior we can ask @mihir174 

https://user-images.githubusercontent.com/2016594/108982934-4ac72000-76c9-11eb-9735-c3cd04557926.mp4